### DISCUSSION TITLE
milvus怎么过滤不相关的内容

### DISCUSSION THREAD

**Discussion by Ping1kl** [milvus-io/milvus#26708] [Q&A and General discussion]
使用text2vec来获取语句的向量，数据放入Milvus 中，查询的时候，给出一个score，我怎么判断这个score，来过滤不相关的内容。例如我问你好吗，Milvus 中没有匹配的数据，就不要返回

**Comment by yhmo** [comment]
它没法知道什么是“不相关”，也没有权值设置让它去判断哪些是“不相关”。所以，只要库里有数据，它永远会尽量返回topk个结果。
